depends/packages/qt.mk

```diff
@@ -263,7 +263,6 @@ endef
 
 define $(package)_config_cmds
   export PKG_CONFIG_SYSROOT_DIR=/ && \
```

Comment:
While we are here, should we also move `PKG_CONFIG_SYSROOT_DIR` into _config_env?